travis-infrastructure/terraform-config
# Use quay.io as registry for the build-env images

## What is the problem that this PR is trying to fix?
Image pulls from Docker Hub stall again, which prevents us from adding new hosts on AWS

## What approach did you choose and why?
Revert to pulling from quay.io.

## How can you test this?
`make clean plan` output:
```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place
-/+ destroy and then create replacement

Terraform will perform the following actions:

-/+ null_resource.language_mapping_json (new resource required)
      id:                          "760804351616841118" => <computed> (forces new resource)
      triggers.%:                  "2" => "2"
      triggers.amethyst_image:     "travisci/ci-amethyst:packer-1503974220" => "quay.io/travisci/ci-amethyst:packer-1503974220" (forces new resource)
      triggers.garnet_image:       "travisci/ci-garnet:packer-1503972846" => "quay.io/travisci/ci-garnet:packer-1503972846" (forces new resource)

  ~ module.aws_asg_com.aws_autoscaling_group.workers
      launch_configuration:        "production-2-workers-com-20171101191943839400000003" => "${aws_launch_configuration.workers.name}"

-/+ module.aws_asg_com.aws_launch_configuration.workers (new resource required)
      id:                          "production-2-workers-com-20171101191943839400000003" => <computed> (forces new resource)
      associate_public_ip_address: "false" => "false"
      ebs_block_device.#:          "0" => <computed>
      ebs_optimized:               "false" => <computed>
      enable_monitoring:           "true" => "true"
      image_id:                    "ami-dddb77a7" => "ami-dddb77a7"
      instance_type:               "c3.8xlarge" => "c3.8xlarge"
      key_name:                    "" => <computed>
      name:                        "production-2-workers-com-20171101191943839400000003" => <computed>
      name_prefix:                 "production-2-workers-com-" => "production-2-workers-com-"
      root_block_device.#:         "0" => <computed>
      security_groups.#:           "4" => "4"
      security_groups.1206244775:  "sg-baf648c9" => "sg-baf648c9"
      security_groups.1507295911:  "sg-9f2860e5" => "sg-9f2860e5"
      security_groups.17498028:    "sg-bbf648c8" => "sg-bbf648c8"
      security_groups.4161445841:  "sg-962860ec" => "sg-962860ec"
      user_data:                   "9bd63d994e98ad22d8b6428fdfe0377a5927b5c8" => "206163112943ad51e1467da5e329a2ea854bdc57" (forces new resource)

  ~ module.aws_asg_org.aws_autoscaling_group.workers
      launch_configuration:        "production-2-workers-org-20171101191942553800000002" => "${aws_launch_configuration.workers.name}"

-/+ module.aws_asg_org.aws_launch_configuration.workers (new resource required)
      id:                          "production-2-workers-org-20171101191942553800000002" => <computed> (forces new resource)
      associate_public_ip_address: "false" => "false"
      ebs_block_device.#:          "0" => <computed>
      ebs_optimized:               "false" => <computed>
      enable_monitoring:           "true" => "true"
      image_id:                    "ami-dddb77a7" => "ami-dddb77a7"
      instance_type:               "c3.8xlarge" => "c3.8xlarge"
      key_name:                    "" => <computed>
      name:                        "production-2-workers-org-20171101191942553800000002" => <computed>
      name_prefix:                 "production-2-workers-org-" => "production-2-workers-org-"
      root_block_device.#:         "0" => <computed>
      security_groups.#:           "2" => "2"
      security_groups.1976880071:  "sg-902860ea" => "sg-902860ea"
      security_groups.2464611514:  "sg-9c2860e6" => "sg-9c2860e6"
      user_data:                   "2a07f603b3a894e514520127ca4b04c1092d0264" => "eacef18bf08027dc5e41d46343646d1eb6a21ab3" (forces new resource)


Plan: 3 to add, 2 to change, 3 to destroy.
```

## What feedback would you like, if any?
Is this safe to `make apply`?
